recOrder/tests/cli_tests/test_cli.py

```diff
@@ -1,10 +1,154 @@
 from recOrder.cli.main import cli
+import unittest
 from click.testing import CliRunner
-
+from unittest.mock import patch, Mock
+from iohub.ngff import open_ome_zarr, Position
+from hypothesis import given, HealthCheck, settings
+from hypothesis import strategies as st
+from recOrder.cli.compute_transfer_function import (
+    generate_and_save_phase_transfer_function, 
+    generate_and_save_birefringence_transfer_function,
+    compute_transfer_function_cli
+)
+from recOrder.cli.settings import (
+    TransferFunctionSettings,
+    _PhaseTransferFunctionSettings,
+    _BirefringenceTransferFunctionSettings,
+    _UniversalSettings
+)
+import pytest
 
 def test_main():
     runner = CliRunner()
     result = runner.invoke(cli)
 
     assert result.exit_code == 0
     assert "Toolkit" in result.output
+
+def test_compute_transfer(tmp_path):
+    runner = CliRunner()
+    result = runner.invoke(cli, "compute-transfer-function -o " + str(tmp_path))
+    assert result.exit_code == 0
+    assert "Generating" in result.output
+
+def test_compute_transfer_blank_config():
+    runner = CliRunner()
+    for option in ("-c ", "--config-path "):
+        cmd = "compute-transfer-function " + option
+        result = runner.invoke(cli, cmd)
+        assert result.exit_code == 2
+        assert "Error" in result.output
+
+def test_compute_transfer_blank_output():
+    runner = CliRunner()
+    for option in ("-o ", "--output-path "):
+        cmd = "compute-transfer-function " + option
+        result = runner.invoke(cli, cmd)
+        assert result.exit_code == 2
```

Comment:
In the previous test case it is already asserted that no `-c` will cause an error. So this test will pass regardless of the behavior (existence of output arg) it is try to test for.